lxl-web/src/routes/(app)/[[lang=lang]]/find/+page.server.ts

```diff
@@ -16,8 +17,10 @@ export const load = async ({ params, url, locals, fetch }) => {
 		redirect(303, `/`); // redirect to home page if no search params are given
 	}
 
+	const debug = locals.userSettings?.debug.includes(DebugFlags.ES_SCORE) ? '&_debug=esScore' : '';
```

Comment:
This is what's causing the test to fail.
Try changing it to
`const debug = locals.userSettings?.debug?.includes(DebugFlags.ES_SCORE) ? '&_debug=esScore' : '';`

Reply:
Because: the test in question is changing the facet sorting, and thereby creating a settings cookie in userSettings. So it has `locals.userSettings` but not `debug`.

Reply:
Yes! That was it. Missed it when changing it from a boolean to an array.